Code/Framework/AzFramework/AzFramework/Viewport/ViewportBus.h

```diff
@@ -77,4 +77,31 @@ namespace AzFramework
     };
 
     using ViewportBorderRequestBus = AZ::EBus<ViewportBorderRequests>;
+
+    //! Provides a bus to notify about different states of the ImGui.
```

Comment:
uber nit: This would be slightly clearer as `Provides an interface to notify listeners about ImGui state changes.`